client/src/game/ui/hypothetical.ts

```diff
@@ -88,11 +88,9 @@ export function send(hypoAction: ClientAction): void {
     case "discard": {
       const card = getCardOrStackBase(hypoAction.target);
       const { suitIndex, rank } = card.getMorphedIdentity();
-      if (suitIndex === null) {
-        throw new Error(`Card ${hypoAction.target} has an unknown suit index.`);
-      }
-      if (rank === null) {
-        throw new Error(`Card ${hypoAction.target} has an unknown rank.`);
+      if (suitIndex === null || rank === null) {
+        // Play or discard action could have been initiated from the keyboard
+        return;
```

Comment:
This seems like the wrong direction. Can we detect this in the keyboard logic and prompt for the desired identity?

Reply:
You mean, like, prevent this in keyboard.ts?

Reply:
Yes, in the same way we prevent in in HanabiCard, by prompting for the desired identity